### DISCUSSION TITLE
About 'parse meta-data from filename'

### DISCUSSION THREAD

**Discussion by nmycps** [Marekkon5/onetagger#206] [Q&A]
On many of my untagged tracks which I'm wanting to populate with the correct meta-data, I do have a good number of them named in a specific format, for example, %title% - %album% - %artist% - %label% - %catalognumber% - %year% - %track#%. I thought that in Auto Tag, enabling 'parse meta-data from filename' would read the filenames as I have written above, but it doesn't seem to. The results just say 'missing artist' or 'missing title'. Do I have to write the above format I use for filename differently in this field for it to be recognized? Thanks for any help.

**Comment by Marekkon5** [comment]
Hello, 

1. I found a bug with the parse filename option, where it only worked with 3 parameters. Now it should work properly. You can get the latest build from Actions tab to test it out. I've tested your prompt and it works.
2. The `%track#%` should be `%track%` or anything, but without special characters, otherwise it will be incorrectly parsed.
3. Please use Issues for bug reports, or feature requests next time, it is easier for us to track. 

Thank you

**Comment by nmycps** [comment]
Hi @Marekkon5 I'm afraid I've not been able to find and run the latest build. Was there an error?

**Reply by Marekkon5** [reply]
There was a problem with pnpm, but should be fixed now.

**Reply by Marekkon5** [reply]
Apparently GitHub Actions have an outage right now, so it will be delayed.

**Reply by Marekkon5** [reply]
Should be done by now.
